.travis.yml

```diff
@@ -37,6 +37,7 @@ matrix:
     - python: 2.7
       sudo: true   # This is set to true for apt-get
       env:
+        - COVERAGE=1
```

Comment:
Should I delete the other ones: lines 26 and 30? That way, we will stop receiving multiple coveralls posts.